Source/WebKit/UIProcess/API/glib/WebKitAuthenticationRequest.cpp

```diff
@@ -75,9 +75,7 @@ struct _WebKitAuthenticationRequestPrivate {
     std::optional<bool> canSaveCredentials;
 };
 
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-static guint signals[LAST_SIGNAL] = { 0, };
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
+static std::array<unsigned, LAST_SIGNAL> signals { };
```

Comment:
I guess with std::array we don't need the `{ } `no?

Reply:
Good point, thanks. I'll update the patch accordingly.